src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx

```diff
@@ -37,7 +41,10 @@ export const AdminSpaceCard: React.FC<AdminSpaceCardProps> = ({ venue }) => {
         <div className={logoClasses} />
         <AdminShowcaseSubTitle>{venue.name}</AdminShowcaseSubTitle>
       </div>
-      <ButtonNG linkTo={adminNGVenueUrl(venue.slug)} disabled={!venue.slug}>
+      <ButtonNG
+        linkTo={adminNGVenueUrl(worldSlug, venue.slug)}
+        disabled={!venue.slug}
```

Comment:
Maybe something like this?
```suggestion
        disabled={!worldSlug || !venue.slug}
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2611/commits/1c9d667fb2f36ca250ca3e965dafc8494c390076